apps/web-giddh/src/assets/css/style-2.scss

```
@@ -1165,12 +1172,12 @@ modal-container.modal {
     color: $nobel;
 }
 .price-input {
-    border: 1px solid $gainsboro;
-    font-size: 14px;
-    background: $white;
-    padding: 1px 2px;
+    border: 1px solid $gainsboro !important;
+    font-size: 14px !important;
+    background: #ffffff !important;
```

Please check color variable change is getting removed.
```suggestion
    background: $white!important;
```

---

Done